docs/docs/sourcing-from-buttercms.md

```diff
@@ -121,11 +121,9 @@ Adding a CMS-powered page to your app involves three easy steps:
 1. Populate the content
 1. Integrate into your application
 
-If you need help after reading this, contact us via email or livechat.
-
 #### Create the page structure
 
-Create a new Page and define it's structure using our Page Builder. Create an example homepage.
+Create a new Page and define its structure using Page Builder. Let's create an example homepage.
```

Comment:
"Let's" is introduced here but it doesn't follow the style guide. Is there another way you can phrase this?

Reply:
I want to make sure this is merged in in time for Hacktoberfest, how about this: 
```suggestion
Create a new Page and define its structure using Page Builder. Create an example homepage to follow along with this guide:
```